fabric-networking-v0/src/main/java/net/fabricmc/fabric/api/network/server/EntityTrackerStreamAccessor.java

```diff
@@ -18,8 +18,14 @@
 
 import java.util.stream.Stream;
 
-import net.minecraft.entity.player.ServerPlayerEntity;
+import com.google.common.annotations.Beta;
 
+import net.minecraft.server.network.ServerPlayerEntity;
+
+/**
+ * Not implemented yet!
+ */
+@Beta
 public interface EntityTrackerStreamAccessor {
```

Comment:
everyday I question who merged this pr to allow these mixin accessors to be in the API package.

Reply:
they're actually meant to be implemented by your entity

Reply:
why it is a thing in the API when it is not done

Reply:
because networking v0 was my first api module and i did not follow api conventions back then